epub-a11y-meta-guide/1.0/draft/index.html

```diff
@@ -961,50 +961,44 @@ <h5>Reading Order</h5>
 						<pre>&lt;meta property="schema:accessibilityFeature"&gt;readingOrder&lt;/meta&gt;</pre>
 					</section>
 
-					<section id="metadata-a11y-features-structuralNavigation">
-						<h5>Structural Navigation</h5>
-
-						<p>The metadata <code>structuralNavigation</code> is used to indicate that an EPUB document has
-							been structured to support accessible navigation through its content. This is crucial for
-							users of assistive technologies.</p>
-
-						<p>The use of headings, lists, tables, asides, etc. in the resource fully and accurately
-							reflects the document hierarchy, allowing navigation by assistive technologies.</p>
-
-						<p>You should use this metadata when your document meets the following criteria:</p>
-
-						<ul>
-							<li><b>Semantic Structure</b>: The document uses semantic HTML5 elements that accurately
-								represent the content's logical structure.</li>
-							<li><b>Heading Hierarchy</b>: Headings (h1-h6) are used in a meaningful, hierarchical manner
-								that reflects the document's outline.</li>
-							<li>
-								<p><b>Navigational Elements</b>: The document includes:</p>
-								<ul>
-									<li>Properly nested headings</li>
-									<li>Ordered and unordered lists where appropriate</li>
-									<li>Tables with appropriate headers and captions</li>
-									<li>Semantic elements like section, article, aside, etc.</li>
-									<li>Meaningful landmark roles</li>
-								</ul>
-							</li>
-						</ul>
-
-						<p>Benefits of Structural Navigation:</p>
-
-						<ul>
-							<li>Enables screen readers to provide better navigation</li>
-							<li>Allows users to jump between sections easily</li>
-							<li>Provides a clear, logical understanding of the document's structure</li>
-							<li>Improves overall accessibility and usability</li>
-						</ul>
-
-						<p>The metadata to include in the package document when proper structural elements are used to
-							facility accessible navigation would be:</p>
-
-						<pre>&lt;meta property="schema:accessibilityFeature"&gt;structuralNavigation&lt;/meta&gt;</pre>
-					</section>
-
+                    <section id="metadata-a11y-features-structuralNavigation">
+                        <h5>structuralNavigation</h5>
```

Comment:
This switches from an English title to the term name.

We could use a different name, too, to avoid the ambiguity of "structural navigation" as it's more like "heading structure in the markup navigation"